app/models/katello/content_view_version.rb

```diff
@@ -447,6 +447,11 @@ def add_applied_filters!
       save!
     end
 
+    def filters_applied?
+      # If applied_filters doesn't exist, return nil
+      self.applied_filters.nil? ? nil : self.applied_filters["applied_filters"].length != 0
```

Comment:
Why not just 
```suggestion
      self.applied_filters.present? 
```

Reply:
+1

Reply:
I guess you want it to be nil for older content view versions where its "unknown"
I wonder if  we can do this

```ruby
def applied_filter_state
    if self.applied_filters.nil? 
       "unknown"
   elsif applied_filters.present?
     "applied"
   else
      "not_applied"
  end
end
```

Reply:
I wasn't sure if this was a hidden feature of `.present?` so I went ahead and tested. For older CVV's that don't contain an applied_filters field, `.present? ` outputs `false` instead of the spec'ed `null`.

Reply:
`self.applied_filters.nil? ? nil : self.applied_filters["applied_filters"].present?` works however

Reply:
How about this
```ruby
   def filters_applied?
       # For older content view versions we do not know if filters were applied.
       # return nil for those. 
       return nil if applied_filters.nil?
       applied_filters.present? 
   end
```

Reply:
That looks great to me, thank you. Just pushed in your suggestions.